### ISSUE TITLE
Error: 'splitOutputLines' is not implemented in Docker Desktop extensions

### ISSUE THREAD

**Issue by RobotSail** [containers/podman-desktop#952]
### Bug description

When providing an options object to `ddClient.docker.cli.exec()`, such as the following:
```ts
ddClient.docker.cli.exec("run", ["hello-world:latest"], { cwd: "/home/user/" })
```
Podman desktop attempts to read the `splitOutputLines` property which isn't defined on the `ExecOptions` interface.
I believe this error occurs in [src/plugin/docker-extension/docker-plugin-adapter.ts:127](https://github.com/RobotSail/podman-desktop/blob/08e987b4c4d447a53f335ea2dccfeae2553e09ee/packages/main/src/plugin/docker-extension/docker-plugin-adapter.ts#L127)

### Operating system

Fedora 35

### Version

next (development version)

### Steps to reproduce

Launch the above command with one of the fields defined in the options object such as:

```ts
ddClient.docker.cli.exec("run", ["hello-world:latest"], { cwd: "/home/user/" })
```

**Comment by benoitf** [comment]
yes, it's not currently implemented.

**Comment by benoitf** [comment]
ok, so looking at the code again, it's because some exec options have been added in the Docker Desktop 0.3.0 API (while we were using 0.2.1 API)
https://docs.docker.com/desktop/extensions-sdk/dev/api/reference/interfaces/ExecOptions/

**Comment by benoitf** [comment]
it's fixed and will be available in 0.11

**Comment by RobotSail** [comment]
Thanks for fixing this @benoitf.
